src/pages/SubmissionDetails.js

```diff
@@ -80,7 +99,11 @@ export const SubmissionDetails = ({ submissionId }) => {
   const specifyRunSet = _.filter(r => r.run_set_id === submissionId, runSetData)
   const methodId = specifyRunSet[0]?.method_id
   const getSpecificMethod = _.filter(m => m.method_id === methodId, methodsData)
-  const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, runsData)
+
+  const errorStates = ['SYSTEM_ERROR', 'EXECUTOR_ERROR']
+  const filteredPreviousRuns = filterOption ? getFilter(filterOption) : runsData
```

Comment:
FYI: if you make `getFilter` more "functional" according to the suggestion above, this could become:

```
const filteredPreviousRuns = filterOption ? getFilter(filterOption)(runsData) : runsData
```